service-registry/src/main/java/io/servicecomb/serviceregistry/config/AbstractPropertiesLoader.java

```diff
@@ -59,9 +59,11 @@ protected void loadPropertiesFromExtendedClass(Configuration configuration, Map<
       extendedPropertyClass = readExtendedPropertyClassName(configuration, EXTENDED_CLASS_FOR_COMPATIBLE);
       if (StringUtils.isEmpty(extendedPropertyClass)) {
         return;
+      } else {
+        LOGGER.warn("The property `{}` will be deprecated soon, please use the new property `{}`",
```

Comment:
maybe it's better to be:
The property `{}` is deprecated and will be removed soon

Reply:
Totally agree. Modifying... @wujimin